docs/authzed/pricing.md

```diff
@@ -0,0 +1,93 @@
+# Pricing on Authzed
+
+[Authzed], the managed permissions database service, has a **usage** based pricing model.
+
+Customers are charged based on two dimensions: the number of [relationships] stored in Authzed, and the number of dispatched operations made when performing [API] calls.
+
+[relationships]: /reference/glossary#relationship
+[API]: /reference/api
+[Authzed]: https://app.authzed.com
+
+| Dimension | Price Per 1 miliion |
+| --------- | ------------------- |
+| Relationships | $1 per 1 million |
+| Dispatched Operations | $1 per 1 million |
```

Comment:
Formatting this table so that the markdown is slightly more readable would be nice, but not required.

Reply:
Sorry, I meant to do that after and forgot